### PR TITLE
[ADD] mrp_lot_cost: MRP Lot Unit Cost

### PR DESCRIPTION
**Pull request by acysos**
Hello,

This module calculates the unit cost of a manufacturing lot taking the costs
of buying lots of raw materials, the value is stored in stock.production.lot
but this thought is not displayed as a tool for other modules

It's useful for products with raw materials like wheat, steel... with a high cost price rotation.

Greetings

### CODE REVIEW

File: mrp_lot_cost/README.rst [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+This module calculates the unit cost of a manufacturing lot taking the costs
+of buying lots of raw materials, the value is stored in stock.production.lot
+but this thought is not displayed as a tool for other modules
+
+Installation
+============
+
+To install this module, you need to:
+
+* clone the branch 8.0 of the repository https://github.com/OCA/manufacture
```
Comment (elicoidal) [inline mrp_lot_cost/README.rst:18]: Thanks for your contribution
I think this is not necessary: I think it is a good idea but it should be consistent with other modules description.

File: mrp_lot_cost/__init__.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# -*- encoding: utf-8 -*-
```
Comment (elicoidal) [inline mrp_lot_cost/__init__.py]: Please use simple headers following OCA templates
https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L3

File: mrp_lot_cost/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    "name": "MRP Lot Unit Cost",
+    "version": "1",
+    "author": "Acysos S.L., Odoo Community Association (OCA)",
+    "website": "www.acysos.com",
+    "contributors": ['Alexander Ezquevo <alexander@acysos.com>', ],
```
Comment (elicoidal) [inline mrp_lot_cost/__openerp__.py]: remove comma

File: mrp_lot_cost/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    "name": "MRP Lot Unit Cost",
+    "version": "1",
```
Comment (elicoidal) [inline mrp_lot_cost/__openerp__.py]: Please follow the guidelines from (8.0.1.0.0):
https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md

File: mrp_lot_cost/__openerp__.py [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+    "author": "Acysos S.L., Odoo Community Association (OCA)",
+    "website": "www.acysos.com",
+    "contributors": ['Alexander Ezquevo <alexander@acysos.com>', ],
+    "category": "Manufacturing",
+    "license": "AGPL-3",
+    "depends": [
+        "mrp",
+    ],
```
Comment (elicoidal) [inline mrp_lot_cost/__openerp__.py:15]: Remove empty keys

File: mrp_lot_cost/models/mrp.py [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+            control = True
+            for line in wiz.consume_lines:
+                if not line.lot_id:
+                    control = False
+            if(control):
+                self.calculateCost(wiz)
+        super(MRP_production, self).action_produce(
+            self.id, production_qty, production_mode, wiz)
+
+    @api.one
```
Comment (rvalyi) [inline mrp_lot_cost/models/mrp.py]: api.one is deprecated in 9. To help forward compatibility please use api.multi + ensure_one
see https://www.odoo.com/documentation/8.0/reference/orm.html#openerp.api.one

File: mrp_lot_cost/README.rst (hunk 1 above)
Comment (acysos) [inline mrp_lot_cost/README.rst:18]: Hello, I don't undertand this comment.

Comment (oihane) [inline mrp_lot_cost/README.rst:18]: If there is not anything "special" to install other than download it and make available, this part is not necessary, so you can erase it.

**Comment by andhit-r** [comment]
Hi @acysos , thanks for your contribution.

CMIIW:
1. Raw material value already carried by stock.quant, no need to add fields on stock.production.lot
2. mrp_production_real_cost can covered functionality of this PR

**Pull request closed without merging**